src/kontrol/prove.py

```diff
@@ -652,66 +657,81 @@ def _method_to_cfg(
-    neworigin_cell = node.cterm.cell('NEWORIGIN_CELL')
-    active_cell = node.cterm.cell('ACTIVE_CELL')
-    depth_cell = node.cterm.cell('DEPTH_CELL')
-    singlecall_cell = node.cterm.cell('SINGLECALL_CELL')
-    gas_cell = node.cterm.cell('GAS_CELL')
-    callgas_cell = node.cterm.cell('CALLGAS_CELL')
-
-    all_accounts = flatten_label('_AccountCellMap_', new_accounts_cell)
-
-    new_accounts = [CTerm(account, []) for account in all_accounts if (type(account) is KApply and account.is_cell)]
-    non_cell_accounts = [account for account in all_accounts if not (type(account) is KApply and account.is_cell)]
-
-    new_accounts_map = {account.cell('ACCTID_CELL'): account for account in new_accounts}
-
-    if config_type == ConfigType.SUMMARY_CONFIG:
-        for account_id, account in new_accounts_map.items():
-            acct_id_cell = account.cell('ACCTID_CELL')
-            if type(acct_id_cell) is KVariable:
-                acct_id = acct_id_cell.name
-            elif type(acct_id_cell) is KToken:
-                acct_id = acct_id_cell.token
-            else:
-                raise RuntimeError(
-                    f'Failed to abstract storage. acctId cell is neither variable nor token: {acct_id_cell}'
-                )
-            new_accounts_map[account_id] = CTerm(
-                set_cell(
-                    account.config,
-                    'STORAGE_CELL',
-                    KVariable(f'STORAGE_{acct_id}', sort=KSort('Map')),
-                ),
-                [],
-            )
+    if config_type == ConfigType.TEST_CONFIG:
+        new_accounts_cell = node.cterm.cell('ACCOUNTS_CELL')
+        number_cell = node.cterm.cell('NUMBER_CELL')
+        timestamp_cell = node.cterm.cell('TIMESTAMP_CELL')
+        basefee_cell = node.cterm.cell('BASEFEE_CELL')
+        chainid_cell = node.cterm.cell('CHAINID_CELL')
+        coinbase_cell = node.cterm.cell('COINBASE_CELL')
+        prevcaller_cell = node.cterm.cell('PREVCALLER_CELL')
+        prevorigin_cell = node.cterm.cell('PREVORIGIN_CELL')
+        newcaller_cell = node.cterm.cell('NEWCALLER_CELL')
+        neworigin_cell = node.cterm.cell('NEWORIGIN_CELL')
+        active_cell = node.cterm.cell('ACTIVE_CELL')
+        depth_cell = node.cterm.cell('DEPTH_CELL')
+        singlecall_cell = node.cterm.cell('SINGLECALL_CELL')
+        gas_cell = node.cterm.cell('GAS_CELL')
+        callgas_cell = node.cterm.cell('CALLGAS_CELL')
+
+        all_accounts = flatten_label('_AccountCellMap_', new_accounts_cell)
+
+        new_accounts = [CTerm(account, []) for account in all_accounts if (type(account) is KApply and account.is_cell)]
+        non_cell_accounts = [account for account in all_accounts if not (type(account) is KApply and account.is_cell)]
+
+        new_accounts_map = {account.cell('ACCTID_CELL'): account for account in new_accounts}
+
+        new_accounts_cell = KEVM.accounts([account.config for account in new_accounts_map.values()] + non_cell_accounts)
```

Comment:
And here update the `cells` dict.
```suggestion
        cells['ACCOUNTS_CELL'] = KEVM.accounts([account.config for account in new_accounts_map.values()] + non_cell_accounts)
```

Reply:
It looks great, thank you @anvacaru! Applied all suggestions in https://github.com/runtimeverification/kontrol/pull/600/commits/78f2894be3cb601b58b7547a2b0ae5cd57bdc925.

Reply:
Update: I realised we need to preserve `new_accounts_cell` variable for the CSE branch to work properly, pushed another small update in https://github.com/runtimeverification/kontrol/pull/600/commits/16cd4e54e0685a4501ce44d113cf5745dd813e60.